lib/mozart_fetcher.ex

```diff
@@ -21,8 +21,23 @@ defmodule MozartFetcher do
     Application.get_env(:mozart_fetcher, :max_connections)
   end
 
-  def cert do
-    System.get_env("DEV_CERT_PEM")
+  def request_ssl do
+    case environment() do
+      :prod -> [certfile: cert_path(), cacertfile: ca_cert_path(), keyfile: key_file_path()]
+      _ -> [certfile: System.get_env("DEV_CERT_PEM")]
+    end
+  end
+
+  defp cert_path do
+    System.get_env("cert_file_path")
```

Comment:
Should we use the same convention for our environment variables, by uppercasing them all?

Reply:
can do, we didn't for requester, and these environment variables are initially set in the cosmos config, and the convention there is for the names to be lower case.

Reply:
Maybe we should use capital letters for the cosmos config keys? https://cosmos.tools.bbc.co.uk/services/ingress/test/configuration we have INSTANCE_ROLE_NAME but also `td_ingest_bucket` which is required to be lower case by the td agent :(

Reply:
yeah that'd be the way to do it, but I'm not fussed either, I think I'll merge this